datascience/README.md

```diff
@@ -52,3 +59,41 @@ Returns :
 3      DE      4229.0  biomass  2016-12-23T00:00:00+00:00
 4      DE     19919.0     coal  2016-12-23T00:00:00+00:00
 ```
+
+#### get_exchange(countries, start_date, end_date)
```

Comment:
@corradio Does that make sense to have a delta parameter for the amount of electricity exchanged ? 
My understanding was that you perform exchange aggregation on the last 24 hours (cf - https://github.com/corradio/electricitymap/blob/master/web/server.js#L175) and then does not really make sense to compute exchange for a smaller time interval.